troubleshooting.md

```diff
@@ -122,17 +122,27 @@ If you are using Mac OS X and see `Unable to retrieve kernel symbols`,
 If your network connection has changed since you loaded the VM, it's possible
 that your IP address is different from that displayed on the Learning VM splash
 screen. Log in to the VM via the virtualization directly (rather than SSH) and
-use the `facter ipaddress` command the check the current address. If you
-continue to get an no IP address or an invalid IP address, restarting the VM is
-generally the quickest way to ensure that the network services are correctly
-reset. (Unfortunately restarting the network service directly isn't always
-reliable.)
+use the `ifconfig | less` command to show the IP address associated with each
+network interface. There will be several interfaces listed, including those
+used for internal docker networking. The correct interface will usually start
+with `eth` or `enp`. There If you continue to get an no IP address or an
+invalid IP address, restarting the VM is generally the quickest way to ensure
```

Comment:
This clause looks like it needs to be reworded: "There If you continue to get an no IP address or an invalid IP address"